backend/check_litellm_routing.py

```
@@ -0,0 +1,225 @@
+def check_litellm_config():
+    """检查LiteLLM配置文件"""
+    print(f"\n📋 检查LiteLLM配置...")
+    
+    config_path = Path(__file__).parent.parent / "litellm" / "config.yaml"
+    
+    if not config_path.exists():
+        print(f"❌ 配置文件不存在: {config_path}")
+        return
+    
+    try:
+        with open(config_path, 'r', encoding='utf-8') as f:
```

Ruff (0.12.2)

Unnecessary mode argument

Remove mode argument

(UP015)